src/server/ua_services_nodemanagement.c

```diff
@@ -811,13 +811,14 @@ addParentAndTypeRef(UA_Server *server, UA_Session *session, const UA_NodeId *nod
 
         if(!server->bootstrapNS0 && node->nodeClass == UA_NODECLASS_OBJECT) {
             if(((const UA_ObjectTypeNode*)type)->isAbstract) {
-                /* Object node created of an abstract ObjectType. Only allowed if within BaseObjectType folder */
+                /* Object node created of an abstract ObjectType. Only allowed
+                 * if within BaseObjectType folder */
                 const UA_NodeId objectTypes = UA_NODEID_NUMERIC(0, UA_NS0ID_BASEOBJECTTYPE);
-                // TODO handle subtypes of parent reference types
```

Comment:
@jpfr Why did you remove this ToDo?
Issue isn't already solved in this code yet nor to add `UA_NS0ID_HASPROPERTY` to the array of possible parent references.